src/Extensions/CommentNotifier.php

```diff
@@ -24,7 +24,7 @@ public function onAfterPostComment(Comment $comment)
     {
         $parent = $comment->Parent();
 
-        if (!$parent) {
+        if (!$parent || !$parent->hasExtension('notificationRecipients')) {
```

Comment:
is this the right name of the extension? it should be a FQCN

Reply:
True is should be hasMethod

Reply:
or `hasExtension(static::class)`

Reply:
Thanks for the feedback. I went for Robbie's suggestion and it fixed the failing unit test too, yeah!